Tasks/DownloadBuildArtifactsV0/task.json

```diff
@@ -102,15 +102,24 @@
       },
       "helpMarkDown": "The build from which to download the artifacts"
     },
+    {
+      "name": "Tags",
+      "type": "string",
+      "label": "Build Tags",
+      "defaultValue": "",
+      "visibleRule": "buildType == specific && buildVersionToDownload != specific",
```

Comment:
this will allow tags to be specified even with buildVersionToDownload == latest. If this is what is needed, update help text for latest also (like how it is updated for latestFromBranch)

Reply:
Isn't this already updated in line 79? If not is there another line I should update?